cluster-autoscaler/cloudprovider/huaweicloud/huaweicloud_service_manager.go

```diff
@@ -62,6 +62,9 @@ type AutoScalingService interface {
 	// IncreaseSizeInstance wait until instance number is updated.
 	IncreaseSizeInstance(groupID string, delta int) error
 
+	// Scale down cluster by updating auto scaling group instances by instanceIDs.
+	ScaleDownCluster(groupID string, instanceIds []string) error
```

Comment:
I'd prefer to take `DeleteScalingInstances(groupID string, instanceIDs []string)` as method name.

The `AutoScalingService` designed to mask implementation details about SDK, but it no `cluster` concept here.

Reply:
Your comments should start with the method name so that it will satisfy `golint` check.